sections/heroes.tex

```diff
@@ -23,7 +23,7 @@ \subsection*{\hypertarget{Secondary}{Secondary Heroes}}
 After Hiring a Secondary Hero, place the model in a Town or Settlement you control.
 \textbf{You can only have one Secondary Hero at a time}.\par
 Secondary Heroes have \textbf{2 Movement Points}; when you gain a Secondary Hero, take an additional set of 2 Movement Tokens to represent their MP.
-They do not have their own Hero Card, \textbf{cannot gain Experience}, and \textbf{cannot use Cards from Your Deck} for any reason.
+They do not have their own Hero Card, \textbf{cannot gain Experience}, and \textbf{cannot play Cards from Your Deck} during Combat.
```

Comment:
```suggestion
They do not have their own Hero Card, \textbf{cannot gain Experience}, and \textbf{cannot play Cards from Your Deck during Combat}.
```
Since it's all about combat, maybe it should be bold too?